src/widgets/training-program/ui/training-program.test.tsx

```diff
@@ -74,7 +37,7 @@ describe('TrainingProgram', () => {
     it('renders Button with correct url', () => {
       const button = screen.getByRole('link', { name: /register/i });
 
-      expect(button).toHaveAttribute('href', 'https://wearecommunity.io/events/rs-angular-2023q4');
+      expect(button).toHaveAttribute('href', '/enroll');
     });
 
     it('renders correct image with alt text', () => {
```

Comment:
Anya asked me to add `data-testid` to pictures and search for it in tests, maybe we should do it here too? Could we also check not only alt but also src here?

Reply:
done